ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -84,7 +81,6 @@ open class ContactView: UIView {
 
     private func setupLayout() {
         var constraints = [NSLayoutConstraint]()
-        constraints.append(widthAnchor.constraint(equalToConstant: Constants.contactWidth))
```

Comment:
Are you removing this constraint because we can just use the content size of the `AvatarView` that each `ContactView` contains (which means that the deleted constraint would be redundant)?

Reply:
yes